fastlane/lib/fastlane/helper/git_helper.rb

```diff
@@ -2,20 +2,24 @@ module Fastlane
   module Actions
     GIT_MERGE_COMMIT_FILTERING_OPTIONS = [:include_merges, :exclude_merges, :only_include_merges].freeze
 
-    def self.git_log_between(pretty_format, from, to, merge_commit_filtering)
-      command = 'git log'
-      command << " --pretty=\"#{pretty_format}\" #{from.shellescape}...#{to.shellescape}"
+    def self.git_log_between(pretty_format, from, to, merge_commit_filtering, date_format = nil)
+      command = ['git log']
+      command << "--pretty=\"#{pretty_format}\""
+      command << "--date=\"#{date_format}\"" if date_format
+      command << "#{from.shellescape}...#{to.shellescape}"
       command << git_log_merge_commit_filtering_option(merge_commit_filtering)
-      Actions.sh(command, log: false).chomp
+      Actions.sh(command.compact.join(' '), log: false).chomp
```

Comment:
`git_log_merge_commit_filtering_option(:include_merges)` will return `nil`. I've amended the method to make this case more clear.